### DISCUSSION TITLE
Support using the new App Store Connect API Key for download_dsyms action

### DISCUSSION THREAD

**Discussion by grEvenX** [fastlane/fastlane#17486] [Ideas]
With the awesome updates to Fastlane it's now possible to perform most of the Fastlane actions using the App Store Connect API Key. It would be nice to see download_dsyms leveraging this too.

**Comment by rogerluan** [comment]
Thanks for the feedback! It's always great to hear the community's needs.

I've added this to my personal to-do list, but can't promise I'll be able to work on this any time soon 🙇 

Pull requests are more than welcome, if you feel like diving into _fastlane_ codebase 🤗 Feel free to tag me to review your PR if you end up opening one!

Thanks!

**Comment by AustinFuller** [comment]
@grEvenX You might find this response [here](https://github.com/fastlane/fastlane/discussions/17485) useful. In short, it currently can't be done until Apple adds support.
